README.rst

```diff
@@ -166,43 +180,49 @@ A `purl` is a URL
 Rules for each `purl` component
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-A `purl` string is an ASCII URL string composed of six components.
+A `purl` string is an ASCII URL string composed of seven components.
 
 Some components are allowed to use other characters beyond ASCII: these
 components must then be UTF-8-encoded strings and percent-encoded as defined in
 the "Character encoding" section.
 
 The rules for each component are:
 
-- **type**:
+- **scheme**:
 
-  - The package `type` is composed only of ASCII letters and numbers, '.', '+'
-    and '-' (period, plus, and dash)
-  - The `type` cannot start with a number
-  - The `type` cannot contains spaces
-  - The `type` must NOT be percent-encoded
-  - The `type` is case insensitive. The canonical form is lowercase
-  - Since a `purl` never contains a URL Authority, its `type` must not be
-    suffixed with double slash as in 'docker://' and should use instead
-    'docker:'. Otherwise this would be an invalid URI per rfc3986 at
+  - The `scheme` is a constant with the value "pkg"
+  - Since a `purl` never contains a URL Authority, its `scheme` must not be
+    suffixed with double slash as in 'pkg://' and should use instead
+    'pkg:'. Otherwise this would be an invalid URI per rfc3986 at
     https://tools.ietf.org/html/rfc3986#section-3.3::
 
         If a URI does not contain an authority component, then the path
         cannot begin with two slash characters ("//").
 
-    While it is acceptable to use such '://' suffix, its is not significant and
-    not needed for unambiguous parsing even if it makes a `purl` look like a
-    familiar web URL. In its canonical form, a `purl` must NOT use such '://'
-    `type` suffix. 
-  - `purl` parsers must accept URLs with such '://' and must ignore the '//'.
+    While it is acceptable to use such '://' scheme suffix, its is not
```

Comment:
In what situations would it be acceptable to use '://' - seems to contradict other statements where it is not allowed

Reply:
Good point and this could be cleaned. The point there was to say that we can make parser cope with undue `://` but using them would make the purl an invalid URL. So to you point It is best to move this part only to the parsing procedure description.
Thanks!